sdk/servicebus/service-bus/test/auth.spec.ts

```diff
@@ -33,9 +35,7 @@ type UnpackReturnType<T extends (...args: any) => any> = ReturnType<T> extends P
 
         const { SERVICEBUS_CONNECTION_STRING: serviceBusConnectionString } = getEnvVars();
 
-        const { Endpoint: fqdn } = parseConnectionString<ServiceBusConnectionStringModel>(
-          serviceBusConnectionString
-        );
+        const fqdn = parseServiceBusConnectionString(serviceBusConnectionString).endpoint;
```

Comment:
Why "fqdn"?
Change to "endpoint"?

Reply:
Ah, that is a mistake. Fixed in https://github.com/Azure/azure-sdk-for-js/pull/11949/commits/5e76e405ea5a394f9f00c08604bbe8c68bcb861b